Sources/epoch_code/compile/functions/EPOCH_worldObjectType.sqf

```diff
@@ -22,6 +22,7 @@
 	Returns:
 	BOOL or ARRAY of BOOLs
 */
+private ["_str","_return","_config","_findStart","_start","_end","_p3dName","_finalConfig","_checkType"];
```

Comment:
Just to note: If a variable is defined in param or params then its already made private so its not need to be listed in the private array but wont hurt either.